docs/admin/user_management.md

```diff
@@ -20,4 +20,10 @@ Users can only be removed if they don't own any teams and projects. As such the
 user should delete any projects and ensure their teams have alternative owners. They
 can either do this themselves or an Admin user can do it for them.
 
-Once all teams and projects are removed, the user can be removed using the option in the "Edit User" dialog.
+### Deleting your own account
```

Comment:
This section is for admin users - not where regular users will go looking for help.

We probably want to add a bit to `docs/cloud/README.md` as well.

Reply:
updated in [b503d52](https://github.com/flowforge/flowforge/pull/1787/commits/b503d521e8a6c2bd9b5fc435c6605106b7a06377)